cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -553,7 +555,16 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 				osbErrors = append(osbErrors, fmt.Errorf("manifest validation error: %v", err))
 			}
 		}
-		osbuildJobResult.JobError = clienterrors.WorkerClientError(clienterrors.ErrorBuildJob, "osbuild build failed", osbErrors)
+
+		reason := "osbuild build failed"
+		if len(failedStages) > 0 {
```

Comment:
It feels a bit awkward to be pluralizing things manually :) You could just use `in stage(s):` instead.

Reply:
As this ends up in the frontend at the customers - I thought it nicer there...
although I really expected that there is a simple library for this (I'm not the golang guy :-/ )

Reply:
Fwiw, this is usually handled via something like https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html as plural forms are complex in different languages. Not sure if we support i18n in the frontend (yet) but even if we don't preparing for it and getting "free" plural handling might be worth it (or not, idk). The topic of what gettext (if we go this route) to use in go is a bit more complex as there is no clear winner (at least last I looked).

Reply:
I'd also prefer a library (not sure which one, though) but I got the feeling we only support English for now.

Reply:
Can there be actually more than one failed stage? I don't think so... :thinking:

Reply:
I wasn't sure if there will be support for running parallel stages soonish - so I'd rather support it here?

Reply:
OK, in that case up to you. I was just looking at the current implementation, which can't return failures in more than one stage.